demo/screencast-dsa-accel-config.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -e
+
+
+# Check if DSA devices exist
+if [ $(ls /dev/dsa/ |wc -l) -lt 2 ] ; then
```

Comment:
if we want to ensure the script does not fail for the user, also the presence of accel-config binary must checked.

Reply:
> we want to ensure the script does not fail for the user
@mythi can you elaborate on this?

As for adding more checks, well we can add plenty of those. We can check if kubectl, curl, asciinema, pv etc are installed, if the proxy is setup correcly, if there are enough disk space for the screencast and so on and so force. The question is do we really want to inflate this simple scripts with tons of checks?

Reply:
no we don't but we started going that path by adding the devices check.  that was my point. I'm fine as it is now.